CONTRIBUTING.md

```diff
@@ -38,7 +38,9 @@ The following points should help guide your development:
   - The build must be deterministic, i.e. a particular commit hash always produces the same build
 - Decentralization: anyone can run the interface
   - An Ethereum node should be the only critical dependency 
-  - All other dependencies should only enhance the UX and [gracefully degrade](https://developer.mozilla.org/en-US/docs/Glossary/Graceful_degradation)
+  - All other external dependencies should only enhance the UX ([graceful degradation](https://developer.mozilla.org/en-US/docs/Glossary/Graceful_degradation))
+- Accessibility: anyone can use the interface
+  - The interface should be responsive and run well on low performance devices
```

Comment:
may be worth emphasizing that a large amount (> 50% of swaps if I remember correctly?) are on mobile.